packages/react-devtools-shared/src/devtools/views/SuspenseTab/SuspenseTab.js

```diff
@@ -85,7 +85,9 @@ function ToggleUniqueSuspenders() {
     <Toggle
       isChecked={uniqueSuspendersOnly}
       onChange={handleToggleUniqueSuspenders}
-      title={'Only include boundaries with unique suspenders'}>
+      title={
```

Comment:
Maybe something like:

"Hide boundaries that are not directly suspended but are showing their fallback" 
or 
"Hide boundaries that are not blocking the reveal of content"

These are probably somewhat inaccurate but maybe the suggestions help :)